components/footer/README.md

```diff
@@ -54,20 +54,10 @@ As read using ChromeVox
 ### HTML
 
 ```html
-<!DOCTYPE html>
-<html lang="en" style="height: 100%;">
-<head>
-    <link href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap-reboot.min.css" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/css?family=Noto+Sans" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
-    <title>Sample Footer</title>
-</head>
-
 <body style="display: flex; flex-direction: column; height: 100vh;">
```

Comment:
Is there a reason we have inline styling?

Reply:
Not sure, it was done by Jason who used to work on the team. I can ask Patrick if there was a specific reason behind it.